From 787752fe6da5235d459ced428891567f6c14b70b Mon Sep 17 00:00:00 2001
From: Xogium <contact@xogium.me>
Date: Thu, 14 Oct 2021 09:40:11 +0200
Subject: [PATCH 09/10] brcmfmac: Increase PMU_MAX_TRANSITION_DLY to 5 seconds

In some cases the AP6236 will take around 3120ms to set the clock,
causing this error:

brcmfmac: brcmf_sdio_htclk: HT Avail timeout (1000000): clkctl 0x50
brcmfmac: brcmf_sdio_htclk: HT Avail timeout (1000000): clkctl 0x50

Wait a little longer before timing out.

This usually happens after cutting power unexpectedly.
---
 drivers/net/wireless/broadcom/brcm80211/brcmfmac/sdio.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/sdio.c b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/sdio.c
index 97ee9e2e2e35..7b0a7f7874b8 100644
--- a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/sdio.c
+++ b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/sdio.c
@@ -311,7 +311,7 @@ struct rte_console {
  * max out at a 1 second (1000000us).
  */
 #undef PMU_MAX_TRANSITION_DLY
-#define PMU_MAX_TRANSITION_DLY 1000000
+#define PMU_MAX_TRANSITION_DLY 5000000
 
 /* Value for ChipClockCSR during initial setup */
 #define BRCMF_INIT_CLKCTL1	(SBSDIO_FORCE_HW_CLKREQ_OFF |	\
-- 
2.33.0

